catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/GmdConstants.java

```diff
@@ -0,0 +1,256 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.spatial.ogc.csw.catalog.common;
+
+import java.util.Arrays;
+import java.util.List;
+
+import javax.xml.namespace.QName;
+
+public class GmdConstants {
+
+    public static final String GMD_REVISION_DATE = "RevisionDate";
+
+    public static final String GMD_ALTERNATE_TITLE = "AlternateTitle";
+
+    public static final String GMD_CREATION_DATE = "CreationDate";
+
+    public static final String GMD_PUBLICATION_DATE = "PublicationDate";
+
+    public static final String GMD_ORGANIZATION_NAME = "OrganisationName";
+
+    public static final String GMD_PREFIX = "gmd";
+
+    public static final String GMD_LOCAL_NAME = "MD_Metadata";
+
+    public static final String GMD_METACARD_TYPE_NAME = GMD_PREFIX + "." + GMD_LOCAL_NAME;
+
+    public static final String GMD_NAMESPACE = "http://www.isotc211.org/2005/gmd";
+
+    public static final String GCO_NAMESPACE = "http://www.isotc211.org/2005/gco";
+
+    public static final String METACARD_URI = "urn:catalog:metacard";
+
+    public static final String APISO_PREFIX = "apiso.";
+
+    public static final String APISO_BOUNDING_BOX = APISO_PREFIX + "BoundingBox";
+
+    public static final String LAST_UPDATE = "lastUpdated";
+
+    public static final String REVISION = "revision";
+
+    public static final String CREATION = "creation";
+
+    public static final String EXPIRY = "expiry";
+
+    public static final String PUBLICATION = "publication";
+
+    public static final String RESOURCE_TYPE = "ext.resource-type";
```

Comment:
Removed this constant in favor of the new Core attribute.  `metadata-content-type` is already being set on the transformer.